controllers/menu_controller.rb

```diff
@@ -1,128 +1,205 @@
+    puts "7 - Exit"
+    print "Enter your selection: "
  
  # #3
-     selection = gets.to_i
-     puts "You picked #{selection}"
-   end
-case selection
-     when 1
-       system "clear"
-       view_all_entries
-       main_menu
-     when 2
-       system "clear"
-       create_entry
-       main_menu
-     when 3
-       system "clear"
-       search_entries
-       main_menu
-     when 4
-       system "clear"
-       read_csv
-       main_menu
-     when 5
-        system "clear"
-        entry_number
-        main_menu
-     when 6
-       puts "Good-bye!"
+    selection = gets.to_i
+    puts "You picked #{selection}"
+  end
+  case selection
+    when 1
+      system "clear"
+      view_all_entries
+      main_menu
+    when 2
+      system "clear"
+      create_entry
+      main_menu
+    when 3
+      system "clear"
+      search_entries
+      main_menu
+    when 4
+      system "clear"
+      read_csv
+      main_menu
+    when 5
+      system "clear"
+      entry_number
+      main_menu
+    when 6
+      system "clear"
+      @address_book.demolish_entries
+      main_menu
+    when 7
+      puts "Good-bye!"
  # #8
-       exit(0)
+      exit(0)
  # #9
-     else
-       system "clear"
-       puts "Sorry, that is not a valid input"
-       main_menu
-     end
-   end
+    else
+      system "clear"
+      puts "Sorry, that is not a valid input"
+      main_menu
+    end
+  end
  
  # #10
    
-def create_entry
+  def create_entry
  # #11
-     system "clear"
-     puts "New AddressBloc Entry"
+    system "clear"
+    puts "New AddressBloc Entry"
  # #12
-     print "Name: "
-     name = gets.chomp
-     print "Phone number: "
-     phone = gets.chomp
-     print "Email: "
-     email = gets.chomp
+    print "Name: "
+    name = gets.chomp
+    print "Phone number: "
+    phone = gets.chomp
+    print "Email: "
+    email = gets.chomp
  
  # #13
-     @address_book.add_entry(name, phone, email)
+    @address_book.add_entry(name, phone, email)
  
-     system "clear"
-     puts "New entry created"
-   end   
+    system "clear"
+    puts "New entry created"
+  end   
 
-   def view_all_entries
+  def view_all_entries
  # #14
-     @address_book.entries.each do |entry|
-     system "clear"
-     puts entry.to_s
+    @address_book.entries.each do |entry|
+    system "clear"
+    puts entry.to_s
  # #15
-       entry_submenu(entry)
-     end
+    entry_submenu(entry)
+  end
  
-     system "clear"
-     puts "End of entries"
+    system "clear"
+    puts "End of entries"
 
-     def entry_submenu(entry)
+  def entry_submenu(entry)
  # #16
-     puts "n - next entry"
-     puts "d - delete entry"
-     puts "e - edit this entry"
-     puts "m - return to main menu"
+    puts "\nn - next entry"
+    puts "d - delete entry"
+    puts "e - edit this entry"
+    puts "m - return to main menu"
  
  # #17
-     selection = gets.chomp
+    selection = $stdin.gets.chomp
  
-     case selection
+  case selection
```

Comment:
indent this case statement it is inside the entry_submenu so should align correctly.